projects/elife.yaml

```diff
@@ -350,6 +350,7 @@ elife-bot:
             "{instance}-elife-published":
                 deletion-policy: retain
                 public: true
+                cors: True
```

Comment:
ah yeah, it's yaml; concidentally everything different from `false` works as truthy, but I'll amend